pkg/controllers/user/controller.go

```diff
@@ -65,16 +72,31 @@ func Register(ctx context.Context,
 			AllowClusterScoped: true,
 		})
 
+	v1alpha1.RegisterUserSyncGeneratingHandler(
+		ctx,
+		userSync,
+		apply,
+		"",
+		"klum-user-sync",
+		h.OnUserSyncChange,
+		&generic.GeneratingHandlerOptions{
+			AllowClusterScoped: true,
+		},
+	)
+
 	secrets.OnChange(ctx, "klum-secret", h.OnSecretChange)
-	user.OnRemove(ctx, "klum-user", h.OnUserRemoved)
+	kconfig.OnChange(ctx, "klum-kconfig", h.OnKubeconfigChange)
+	userSync.OnRemove(ctx, "klum-usersync", h.OnUserSyncRemove)
```

Comment:
Should these two differ? `klum-user-sync` vs `klum-usersync`?

Reply:
it doesn't really matter but changed it for consistence